### DISCUSSION TITLE
Installation problem with Elpaca

### DISCUSSION THREAD

**Discussion by ronspe** [alphapapa/activities.el#127] [Q&A]
Hi,

I currently have used Burly, and I want to try out Activities.
I use the elpaca package manager but that will not let me install "Activities" from the repository. (I have installed Burly with elpaca without any problems.)
So I have copied the three activities*.el files to a new folder in .emacs.d and try to install it manually.

This seems to load correctly, I can see the commands after Mx but when I try to define or create an activity I get the message: "cannot open load file: no such file or directory, activities"
The user-rights on the "activities" folder  are read+write for all users.
This is my configuration: 

```
(use-package activities
    :load-path "manual-packages/activities/" 
    :elpaca nil
    :init
    (activities-mode)
    (activities-tabs-mode)
    ;; Prevent `edebug' default bindings from interfering.
    (setq edebug-inhibit-emacs-lisp-mode-bindings t)
    :bind
    (("C-x C-a C-n" . activities-new)
     ("C-x C-a C-d" . activities-define)
     ("C-x C-a C-a" . activities-resume)
     ("C-x C-a C-s" . activities-suspend)
     ("C-x C-a C-k" . activities-kill)
     ("C-x C-a RET" . activities-switch)
     ("C-x C-a b" . activities-switch-buffer)
     ("C-x C-a g" . activities-revert)))
```

What can I do to get rid of the error message?

**Comment by alphapapa** [comment]
Please see the installation instructions: https://github.com/alphapapa/activities.el#installation
